index.bs

```diff
@@ -1077,15 +1101,23 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
           </ol>
         </li>
         <li>
-          Let <var>p</var> be a new promise.
+          Let <var>p</var> be a new Promise.
         </li>
         <li>
-          <a>In parallel</a>, run the <a>Create a
-          MediaCapabilitiesDecodingInfo</a> algorithm with
-          <var>configuration</var> and resolve <var>p</var> with its result.
+          Return <var>p</var>.
         </li>
         <li>
-          Return <var>p</var>.
+          <a>In parallel</a>, run following steps:
```

Comment:
Missing "the", and for consistency with the other occurrence, I would suggest

```suggestion
          Run the following steps <a>in parallel</a>:
```

Reply:
Done